botocore/client.py

```diff
@@ -15,6 +15,7 @@
 
 import botocore.serialize
 import botocore.validate
+import botocore.parsers
```

Comment:
Added as `botocore.parsers` is referenced further down in:
`response_parser = botocore.parsers.create_parser(protocol)`
but was not defined as it was never imported.